yy_parser.go

```diff
@@ -65,6 +65,11 @@ func TrimComment(txt string) string {
 	return specCodeEnd.ReplaceAllString(txt, "")
 }
 
+type ParserConfig struct {
+	EnableWindowFunction  bool
+	StrictDoubleTypeCheck bool
```

Comment:
How about renaming it to `StrictDoubleTypeCheck`?

Reply:
@bb7133 it is `StrictDoubleTypeCheck`

Reply:
@blacktear23, he meaned `EnableStrictDoubleTypeCheck`.

Reply:
No problem

Reply:
:)